generate.sh

```diff
@@ -129,7 +130,11 @@ fi
 
 if [ ! -f index.rst ]; then
   echo -e "No index.rst found. Auto generating...\n"
-  python $BASE/modules/scripts/genindex.py -s $SUBPROJECT_DIRS -d $SUBPROJECT_DOCPATHS $ROOT_DIR
+  GENINDEX_PARAM=$ROOT_DIR
+  if [ "$DOCPATH" == "." ]; then
+    GENINDEX_PARAM=$ROOT_DIR/yaydoctemp
+  fi
+  python $BASE/modules/scripts/genindex.py -s "$SUBPROJECT_DIRS" -d "$SUBPROJECT_DOCPATHS" "$GENINDEX_PARAM"
```

Comment:
Added quotes around parameters so that they are correctly passes when variables are empty